Fix stock search (#76)
xander-deanna/project-1-group-7: README.md

_Tradify is a cutting edge investment tool for today's investors._

> _Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who wantpeer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.

Author: ```suggestion
_Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who want to peer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.
```

_Tradify is a cutting edge investment tool for today's investors._

_Motivated by the recent financial revolution happening in US markets, we wanted to develop a tool that empowers investors of all types, levels, and interest. With Tradify, savvy investors who want to peer into the world of stocks, cryptocurrency, and foreign exchange markets, can access our platform to get up to the minute pricing data and insight in three financial markets in one tool, instead of having to use a separate tool for each.

> This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/) and we also allow users to search current cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._

Author: ```suggestion
This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/), and cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._
```

> This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/) to and we also allow users to search current cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._

Author: ```suggestion
This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/) and we also allow users to search current cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._
```

> This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/) to and we also allow users to search current cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._

Author: ```suggestion
This app will run in the browser and feature dynamically updated HTML and CSS powered by JavaScript, and uses the Foundation CSS framework for styling HTML. Users can retrieve current stocks using [Stock API](https://www.alphavantage.co/documentation/) , and cryptocurrency prices using [Coinbase API](https://developers.coinbase.com/)._
```

## **Technologies**

> _HTML5, Foundation, CSS, Javascript, Fetch API, AlphaVantage API, Coinbase API_

Author: ```suggestion
_HTML5, Foundation, CSS, Javascript, AlphaVantage API, Coinbase API_
```
